### PR TITLE
Added retrieval of scoreboard

### PR DESCRIPTION
**Pull request by tuupke**
The to string method can use some work. But at least the types are in.

### CODE REVIEW

File: scoreboard.go [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+		EventId     Identifier `json:"event_id"`
+		Time        ApiTime    `json:"time"`
+		ContestTime ApiRelTime `json:"contest_time"`
+		State       State      `json:"state"`
+		Rows        []Row      `json:"rows"`
+	}
+
+	State struct {
+		Started      ApiTime `json:"started"`
+		Ended        ApiTime `json:"ended"`
```
Comment (nickygerritsen) [inline scoreboard.go:20]: These state times can be empty. Then they should be pointers right?

Comment (tuupke) [reply scoreboard.go:20]: I think I misread the table, all of them should be pointers indeed. Thawed also should be omitempty since it is not required.

**Comment by deboer-tim** [comment]
Since we have the State info we should probably just expose the /state endpoint directly later too.

**Comment by tuupke** [comment]
> Since we have the State info we should probably just expose the /state endpoint directly later too.

I've added it, though now the formatting of the text is different from the other types. Due to the extra indentation needed for the formatting of the scoreboard.

**Review by nickygerritsen — APPROVED** [review]

**Pull request merged**